### ISSUE TITLE
Windows: certificate is installed to Intermediate Certification Authorities

### ISSUE THREAD

**Issue by 3052** [mitmproxy/mitmproxy#6523]
If I install `mitmproxy-ca-cert.cer` on Windows using the default option "Automatically select the certificate store based on the type of certificate", the certificate gets installed to "Intermediate Certification Authorities", which is not valid for intercepting requests.

Instead, I need to manually choose to install to Trusted Root Certification Authorities. not sure if its possible, but I think it would be helpful if the certificate was made in such a way that it defaulted to being installed into the correct location.

**Comment by mhils** [comment]
I don't think this is possible, but of course I'd appreciate any pointers!

**Comment by mhils** [comment]
https://docs.mitmproxy.org/stable/concepts-certificates/ has something, but
that's outdated. Contributions welcome!

**Comment by 3052** [comment]
ah thats pretty cool. what is the reason for recommending `mitmproxy-ca-cert.p12` over the `.cer` file?

**Comment by Anu-Ra-g** [comment]
The mitmproxy docs include a way to manually install the certificate inside "Trusted Root Certification Authorities".

**Comment by Anu-Ra-g** [comment]
I fail to understand that only for Windows, we are referring to two different certificates for manual installation(mitmproxy-ca-cert.p12) and automatic installation(mitmproxy-ca-cert.cer). Is it necessary?

**Comment by 3052** [comment]
yeah I always thought that was confusing - I usually prefer the PEM since its text format - but the CER is nice too because you get the GUI that parses the info

**Comment by anurag-waterdip** [comment]
The automated installation of the certificate show me this error.
<img width="614" alt="Screenshot 2024-02-10 094414" src="https://github.com/mitmproxy/mitmproxy/assets/153485246/81edded7-0cee-448c-ad18-6650224ab378">

**Comment by Anu-Ra-g** [comment]
@3052 how are you able to install the .cer file when you're installing a .p12 file?
<img width="614" alt="Screenshot 2024-02-10 094414" src="https://github.com/mitmproxy/mitmproxy/assets/94440065/dfef1488-217e-44d1-a90f-404fcf94e0ba">

**Comment by 3052** [comment]
I dont touch the `.p12` files. I just right click the `.cer` file and select Install Certificate

**Comment by Anu-Ra-g** [comment]
On mitm.it, there is no link to the .cer file!!

**Comment by 3052** [comment]
I dont use `mitm.it` with Windows. when you run `mitmproxy`, it generates all the files in the user folder.

however I would agree with you, only listing the `.p12` on the website, or even recommending over other options seems dumb. the `mitm.it` is really in need of improvement

**Comment by Anu-Ra-g** [comment]
@3052 I tried out what you are facing. According to you what all changes should be made to the documentation including the mitm.it page, keeping the above discussion in consideration. I'd like to contribute to the mitmproxy's documentation.
